reconcile/slack_usergroups.py

```diff
@@ -243,6 +243,11 @@ def get_desired_state(slack_map, pagerduty_map):
         workspace_name = workspace['name']
         usergroup = p['handle']
         description = p['description']
+        channel_names = [] if p['channels'] is None else p['channels']
+        if channel_names == []:
+            # This is to get pass the app-sre-escalation-undefined
+            # usergroup in placeholder-escalation-permission.
```

Comment:
nit: maybe worth adding a DEBUG log message here? It never hurts to have more information if someone is troubleshooting unexpected conditions.

Reply:
+1